app/src/main/java/com/android/doctorapp/ui/doctor/AddDoctorViewModel.kt

```diff
@@ -179,14 +197,13 @@ class AddDoctorViewModel @Inject constructor(
 
     fun validateAllUpdateField() {
         if (isDoctor.value!!) {
-
             isUpdateDataValid.value = (!name.value.isNullOrEmpty() && !email.value.isNullOrEmpty()
                     && !contactNumber.value.isNullOrEmpty() && nameError.value.isNullOrEmpty()
                     && emailError.value.isNullOrEmpty() && contactNumberError.value.isNullOrEmpty()
                     && !address.value.isNullOrEmpty() && addressError.value.isNullOrEmpty()
                     && !dob.value.isNullOrEmpty() && dobError.value.isNullOrEmpty()
-                    && !isAvailableDate.value.isNullOrEmpty() && isAvailableDateError.value.isNullOrEmpty()
-                    && !availableTime.value.isNullOrEmpty() && availableTimeError.value.isNullOrEmpty()
+                    && availableTimeList.value?.isEmpty() == false && availableTimeError.value.isNullOrEmpty()
```

Comment:
availableTimeList.value?.isEmpty() == false you can use like this way also

availableTimeList.value.isNotEmpty()

Reply:
Done